Pickers/AbstractActionSheetPicker.m

```diff
@@ -252,6 +252,13 @@ - (void)showActionSheetPicker {
             height = [datePicker getDatePickerHeight];
         }
     }
+    
+    /// Bottom padding for iPhone X style phones (adds some additional height for the home bar).
+    if (@available(iOS 11.0, *)) {
+        UIWindow *window = UIApplication.sharedApplication.keyWindow;
+        height += window.safeAreaInsets.bottom;
+    }
```

Comment:
`keyWindow` is deprecated
https://stackoverflow.com/questions/57134259/how-to-resolve-keywindow-was-deprecated-in-ios-13-0

fyi @skywinder @palmtrae

Reply:
True, there's more examples of this in the code-base. Should probably open a new issue with this in mind to add scene support for iPads.

Reply:
Agree. It would be nice if someone can deal with is.